src/onnx/parse_constant.cpp

```diff
@@ -39,16 +39,29 @@ struct parse_constant : op_parser<parse_constant>
                           onnx_parser::node_info info,
                           const std::vector<instruction_ref>& /*args*/) const
     {
-        literal v = parser.parse_value(info.attributes.at("value"));
+        static const std::string attributes[] = {
+            "value", "value_float", "value_floats", "value_int", "value_ints"};
+
+        auto&& attr = [&] {
+            for(const auto& a : attributes)
+            {
+                if(auto it = info.attributes.find(a); it != info.attributes.end())
+                {
+                    return it->second;
+                }
```

Comment:
style: use `contains(C, a)` instead from `ranges.hpp`

Reply:
Noted, the code uses it now.